hazelcast/src/main/java/com/hazelcast/config/RingbufferStoreConfig.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.core.RingbufferStore;
+import com.hazelcast.core.RingbufferStoreFactory;
+
+import java.util.Properties;
+
+import static com.hazelcast.util.Preconditions.isNotNull;
+
+/**
+ * Configuration for the {@link RingbufferStore}.
+ */
+public class RingbufferStoreConfig {
```

Comment:
Is there a ReadonlyRingbufferStoreConfig? Normally there is some kind of immutable wrapper around the config.